packages/block-library/src/accordion/block.json

```
@@ -62,5 +62,6 @@
 	},
 	"allowedBlocks": [ "core/accordion-content" ],
 	"textdomain": "default",
-	"style": "wp-block-accordion"
+	"style": "wp-block-accordion",
+	"viewScriptModule": "file:./view.js"
```

This should be able to enqueue the Module ID:

```suggestion
	"viewScriptModule": "@wordpress/block-library/accordion/view"
```

(EDIT: I suggested `@wordpress/block-library/accordion` from the original PR which is not the module ID I'd expect. I've corrected the suggestion.)

These should be registered based on this package.json field:

https://github.com/WordPress/gutenberg/blob/99d25fb6620bbf08d65bd537473758e29922bfca/packages/block-library/package.json#L29-L37

So `./accordion/view` becomes `@wordpress/block-library/accordion/view`.